src/main/java/io/stargate/sgv2/jsonapi/service/resolver/update/TableUpdateResolver.java

```diff
@@ -86,6 +93,20 @@ public WithWarnings<UpdateValuesCQLClause> resolve(
       assignments.addAll(resolverFunction.apply(commandContext.schemaObject(), arguments));
     }
 
+    if (!missingAssignmentsOperators.isEmpty()) {
```

Comment:
I am not sure this is the solution we want, this looks like it would throw is the user set a `$set : {}` and `$unset: {"column": ""} which I think should be legal. 

Feels like the test should be more that we should have atleast one assignment, and the above is OK 

Lets talk, and if that is OK that we can perhaps re-use the MISSING_UPDATE_OPERATIONS error code above and make the message say you need to send atleast one assignment